templates/default/elasticsearch.init.erb

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
```

Comment:
It does seem like most cookbooks end up providing an init script, unfortunately.

> Adding just a really minimal, stupid init script and encourage people provide their own (something really simple like Nginx has?

That seems like a good way to go.